src/Network/HaskellNet/SSL.hs

```diff
@@ -3,7 +3,12 @@ module Network.HaskellNet.SSL
   , defaultSettingsWithPort
   ) where
 
+#if defined(NETWORKLESS03)
```

Comment:
Where does this `NETWORKLESS03` flag come from? It is not defined in the cabal file.

Did you mean to use this standard flag (with reversed logic)?
`#if MIN_VERSION_network(3,0,0)`